generate-function.js

```diff
@@ -61,13 +62,13 @@ const genfun = function() {
       })
       .join('\n')
 
-    return `(function() {\n${scopeSource}\nreturn (${line})})();`
+    return `(function() {\n${scopeSource}\nreturn (${builder.toString()})})();`
```

Comment:
`.toString()` is implied inside string substitution
```suggestion
    return `(function() {\n${scopeSource}\nreturn (${builder})})();`
```

Reply:
I am aware, but I don't like the implicit conversion of the builder object to a string here.
Perhaps we should name the method something else, not `.toString`, not sure.